src/task/task-orchestra.ts

```diff
@@ -0,0 +1,58 @@
+import { inject, injectable } from 'inversify';
+import { ConfigLoader, Scraper, TYPES } from '../types';
+import { CommonTask, Task, TaskType } from './task-types';
+import Timeout = NodeJS.Timeout;
+
```

Comment:
Better add some document in the comment to describe the purpose and design of this class

Reply:
Will do in a separated PR